pkg/ccl/kvccl/kvfollowerreadsccl/followerreads.go

```diff
@@ -203,6 +205,49 @@ func (o *followerReadOracle) useClosestOracle(
 // whether a query can be used with a follower read.
 var followerReadOraclePolicy = replicaoracle.RegisterPolicy(newFollowerReadOracle)
 
+type bulkOracle struct {
```

Comment:
I called it "bulk" based on its usage rather than its behavior since I didn't want to be inaccurate about its behavior if I gave it behavior-based name. I worry I will be inaccurate if I just call it uniform or something, particularly since I hope to in the future teach this thing to be "sticky" to a given node _up to a point_ for adjacent spans.